test/capwap_loc_provider_SUITE.erl

```diff
@@ -0,0 +1,170 @@
+%% Copyright (C) 2013-2023, Travelping GmbH <info@travelping.com>
```

Comment:
this is new code, it should therefore be (C) 2023 only

Reply:
ok